test/unified-test-format/invalid/collectionData-documents-type.json

```diff
@@ -18,8 +18,7 @@
       "collection": {
         "id": "collection0",
         "database": "database0",
-        "collectionName": "foo",
```

Comment:
Are these "foo" spec changes intentional?

Reply:
Yes these were changed in the spec repo.